src/utils.js

```diff
@@ -32,3 +39,99 @@ export function numberToDaysOfWeek(num) {
 export function daysOfWeekToNumber(days) {
   return days.reduce((acc, day) => acc + 2 ** DAYS_OF_WEEK.indexOf(day), 0);
 }
+
+/**
+ * Check if job title include search query
+ * @param {Object} job
+ * @param {String} search
+ */
+function searchCheck(job, search) {
+  if (!job) return false;
+  return (
+    job.title.toLowerCase().includes(search.toLowerCase()) ||
+    job.company.name.toLowerCase().includes(search.toLowerCase())
+  );
+}
+
+/**
+ * Check for jobPosted field of job
+ * @param {Object} job
+ * @param {String} jobPosted (index of ["Anytime", "Past 3 days", "Past week", "Past month"])
+ */
+function jobPostedCheck(job, jobPosted) {
+  if (!job) return false;
+  //todo: write the function
+  return true;
+}
+
+/**
+ * Check for jobType field of job
+ * @param {Object} job
+ * @param {String[]} jobType (indices of ["Part-time", "Full-time", "Shift based"])
+ */
+function jobTypeCheck(job, jobType) {
+  if (!job) return false;
+  return (
+    !jobType.length || jobType.map((id) => JOB_TYPE[parseInt(id)]).includes(job.jobType)
+  );
+}
+
+/**
+ * Check for wage field of job
+ * @param {Object} job
+ * @param {String} wage ([0, 5, 10, 18, 25, 32])
+ */
+function wageCheck(job, wage) {
+  if (!job) return false;
+  if (job.wage.type === "Yearly") {
```

Comment:
I see. @KiLiG-fcx  and you will both work on these fields (job type and salary type. 
To avoid conflict, I will create a PR by my side to define the constants here. Could you take a look at the new PR, and apply the constants then?
Also, this may require some changes for you code here. But it would very simple.